api/app/utils/review.inference.js

```diff
@@ -63,9 +63,9 @@ async function calculateWordDiff(data) {
   const fieldsToConsider = [
     "title",
     "transcript_by",
-    "categories",
-    "tags",
-    "speakers",
+    // "categories",
```

Comment:
Why are these fields commented out?

Reply:
Was commented when debugging what field are causing the error